conda/base/context.py

```diff
@@ -111,6 +127,8 @@ class Context(Configuration):
     _custom_multichannels = MapParameter(Sequence, aliases=('custom_multichannels',))
 
     # command line
+    default_python = PrimitiveParameter(default_python_default(),
+                                        validation=default_python_validation)
```

Comment:
Do we need to add this config?  I see it's only used right now in a test.  It might be useful, but I want to make sure we're thinking about this in the context of a 3 months from now when the majority of new miniconda installs will be putting conda in a private env, with it's own private python.  In that case there won't really be this concept of "use the same python version in my newly created env as exists in my root env."